### ISSUE TITLE
Documentation Improvement

### ISSUE THREAD

**Issue by schiederme** [peering-manager/peering-manager#310]
I tried and failed to setup Peering Manager according to the documentation.

I currently work on improving the documentation.

I open this issue for discussion.

Things to consider:
- I added steps and tested for Ubuntu 20.04 and will test CentOS.
- Place "migrating to psql" into a new page

Im willing to invest some time to make a proper and multi os documentation (like netbox).

**Comment by respawner** [comment]
You can consider removing the old sqlite to psql stuff. This was for early users, before 1.0. If someone still run a version with sqlite as database, he/she should probably consider starting from scratch 🙂

**Comment by popanonymous** [comment]
I'm not having much luck as well.  Believe it's permission/gunicorn related. 

I am able to bring up the application using this:
  python3 manage.py runserver 0.0.0.0:8000 --insecure

But when I attempt to start the peering-manager-rq service it says (courtesy of journalctl -eu) 
  ModuleNotFoundError: No module named 'django'

Any pointers for a newb?  I have installed netbox in the past which follows the same flow, more or less.

**Comment by popanonymous** [comment]
So I figured it out.
Run thru install as root.
There is a couple things not called out in requirements, I want to say PIP, but not sure.  
  - I'm going to do this a 4th time and document the deltas.
 
The peering-manager.service file has a typo.  Should be peering_manager.wsgi and not peering-manager.wsgi.  I changed that line to match the previous test and I was able to run Exec-Start succesfully from bash.  Once I was able to run fine from there, updated the file, killed process, restarted and it's good.
